### ISSUE TITLE
Some questions about the x and y coordinates!

### ISSUE THREAD

**Issue by FightStone** [princeton-vl/pose-ae-train#44]
1. In calc() function which in group.py file, x and y define for this:
    x = ind % w
    y = (ind / w).long()
    ind_k = torch.stack((x, y), dim=3)
2. But in adjust() function which in group.py file, the order of x and y is:
    y, x = joint[0:2]
3. And in refine() function which in test.py file, the order of x and y is:
    y, x = keypoints[i][:2].astype(np.int32)
4. Bue in genDtByPred() function which in test.py file, order of x and y is:
    tmp["keypoints"] += [float(j[0]), float(j[1]), 1]

**Comment by FightStone** [comment]
Who can help me answer the question? and Is the code correct?？ Thanks a lot !
